templates/jlink-interim-build-template.yaml

```diff
@@ -0,0 +1,82 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  annotations:
+    description: Template to produce an imagestream and buildconfig for a Jlinked intermediate image
+  name: s2i-build-template
+parameters:
+- name: APP_NAME
+  description: Name of the application
+  required: true
+- name: GIT_URI
+  description: Git repository URL
+  required: true
+- name: GIT_REF
+  description: Git branch or tag to build
+  required: true
+- name: CONTEXT_DIR
+  description: Context directory which is a directory within git repo
+  required: true  
+- name: BUILDER_IMAGE
```

Comment:
i.e. the output of phase1: builder image + jmods RPM. `ubi9-openjdk-${JDK_VERSION}-jlink`.

Reply:
sure